src/rt/cover.c

```diff
@@ -175,6 +175,37 @@ static bool cover_is_branch(tree_t branch)
    return tree_kind(branch) == T_ASSOC || tree_kind(branch) == T_COND;
 }
 
+unsigned cover_get_std_log_expr_flags(tree_t decl)
+{
+   assert(tree_kind(decl) == T_FUNC_DECL);
+
+   ident_t ident = tree_ident(decl);
+   ident_t ident_2 = tree_ident2(decl);
+
+   if (!ident_starts_with(ident_2, ident_new("IEEE.STD_LOGIC_1164")))
+      return 0;
+
+   // TODO: Should we create this array somewhere in advance ?
+   struct {
+      ident_t op;
+      unsigned flags;
+   } std_log_ops[] = {
+      { ident_new("\"and\"")  ,COVER_FLAGS_AND_EXPR},
+      { ident_new("\"nand\"") ,COVER_FLAGS_AND_EXPR},
+      { ident_new("\"or\"")   ,COVER_FLAGS_OR_EXPR},
+      { ident_new("\"nor\"")  ,COVER_FLAGS_OR_EXPR},
+      { ident_new("\"xor\"")  ,COVER_FLAGS_XOR_EXPR},
+      { ident_new("\"xnor\"") ,COVER_FLAGS_XOR_EXPR}
```

Comment:
You can add these to `intern_strings()` in `common.c` and then make the first line of the table a `well_known_t`.